modules/restful_example/src/Plugin/resource/Articles__1_0.php

```diff
@@ -39,42 +39,44 @@ class Articles__1_0 extends ResourceEntity implements ResourceInterface {
    * {@inheritdoc}
    */
   protected function publicFields() {
-    return array(
-      'id' => array(
-        'property' => 'nid',
-      ),
-      'label' => array(
-        'wrapper_method' => 'label',
-        'wrapper_method_on_entity' => TRUE,
-        'process_callbacks' => array(
-          array(array($this, 'addPrefix'), array('Label: ')),
-        ),
-      ),
-      'self' => array(
-        'callback' => array($this, 'getEntitySelf'),
-      ),
-      'tags' => array(
-        'property' => 'field_tags',
-        'resource' => array(
-          'name' => 'tags',
-          'majorVersion' => 1,
-          'minorVersion' => 0,
-        ),
+    $public_fields = array();
+    $public_fields['id'] = array(
+      'property' => 'nid',
+      'methods' => array(RequestInterface::METHOD_GET),
```

Comment:
It can be moved to `ResourceEntity`, but I am not sure I see the value of doing so.